lstchain/io/lstcontainers.py

```diff
@@ -51,23 +56,34 @@ class DL1ParametersContainer(Container):
     disp_miss = Field(None, 'disp_miss [m]', unit=u.m)
     src_x = Field(None, 'source x coordinate in camera frame', unit=u.m)
     src_y = Field(None, 'source y coordinate in camera frame', unit=u.m)
-    time_gradient = Field(None, 'Time gradient in the camera')
-    intercept = Field(None, 'Intercept')
-    leakage_intensity_width_1 = Field(None, 'Fraction of intensity in outermost pixels')
-    leakage_intensity_width_2 = Field(None, 'Fraction of intensity in two outermost rings of pixels')
-    leakage_pixels_width_1 = Field(None, 'Fraction of signal pixels that are border pixels')
-    leakage_pixels_width_2 = Field(None, 'Fraction of signal pixels that are in the two outermost rings of pixels')
-    n_pixels = Field(None, 'Number of pixels after cleaning')
-    concentration_cog = Field(None, 'Fraction of intensity in three pixels closest to the cog')
-    concentration_core = Field(None, 'Fraction of intensity inside hillas ellipse')
-    concentration_pixel = Field(None, 'Fraction of intensity in brightest pixel')
-    n_islands = Field(None, 'Number of Islands')
-    alt_tel = Field(None, 'Telescope altitude pointing', unit=u.rad)
-    az_tel = Field(None, 'Telescope azimuth pointing', unit=u.rad)
-
-    obs_id = Field(None, 'Observation ID')
-    event_id = Field(None, 'Event ID')
-    calibration_id = Field(None, 'ID of the employed calibration event')
+    time_gradient = Field(np.nan, 'Time gradient in the camera')
+    intercept = Field(np.nan, 'Intercept')
+    leakage_intensity_width_1 = \
+        Field(np.float32(np.nan), 'Fraction of intensity in outermost pixels',
+              dtype=np.float32)
+    leakage_intensity_width_2 = \
+        Field(np.float32(np.nan), 'Fraction of intensity in two outermost '
+                                  'rings of pixels', dtype=np.float32)
+    leakage_pixels_width_1 = Field(np.nan, 'Fraction of signal pixels that are '
+                                           'border pixels')
+    leakage_pixels_width_2 = Field(np.nan, 'Fraction of signal pixels that are '
+                                           'in the two outermost rings of pixels')
+    n_pixels = Field(0, 'Number of pixels after cleaning')
+    concentration_cog = Field(np.nan, 'Fraction of intensity in three pixels '
+                                      'closest to the cog')
+    concentration_core = Field(np.nan, 'Fraction of intensity inside hillas '
+                                       'ellipse')
+    concentration_pixel = Field(np.nan, 'Fraction of intensity in brightest '
+                                        'pixel')
+    n_islands = Field(0, 'Number of Islands')
```

Comment:
for events that do not pass analysis cuts, this may actually be 0, right? I think it would be better if we set it to -1

Reply:
An event that has 0 pixels after cleaning has, well, 0 pixels and 0 islands.